src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs

```diff
@@ -129,12 +129,10 @@ public CdnjsCatalog(CdnjsProvider provider)
         {
             try
             {
-                string[] args = libraryId.Split('@');
-                string name = args[0];
-                string version = (args.Count() > 1) ? args[1] :null;
+                string name = _provider.GetLibraryName(libraryId);
+                string version = _provider.GetLibraryVersion(libraryId);
 
                 IEnumerable<Asset> assets = await GetAssetsAsync(name, cancellationToken).ConfigureAwait(false);
-
                 Asset asset = assets.FirstOrDefault(a => a.Version == version);
```

Comment:
>FirstOrDefault [](start = 37, length = 14)

might as well just do a .First(...) and have the exception thrown on your behalf and delete the asset == null check below.